src/services/EdFi.DataManagementService.Api/Infrastructure/Extensions/HttpRequestResponseExtensions.cs

```diff
@@ -16,8 +16,7 @@ public static string RootUrl(this HttpRequest httpRequest)
 
     public static string UrlWithPathSegment(this HttpRequest httpRequest)
     {
-        var rootUrl = $"{httpRequest.Scheme}://{httpRequest.Host}{httpRequest.PathBase}";
-        return $"{rootUrl}{httpRequest.Path}";
+        return $"{httpRequest.RootUrl()}{httpRequest.Path}".TrimEnd('/');
```

Comment:
Before, we ended up getting different URL's in the JSON if the user entered a slash at the end or omitted it. Now, we always treat as if the slash were omitted.